src/Kovan-DssSpell.sol

```diff
@@ -22,271 +22,59 @@ import "dss-exec-lib/DssAction.sol";
-        DssExecLib.updateCollateralPrice(ilk);
-
-        // bump changelog version
-        DssExecLib.setChangelogVersion("1.9.2");
+    // Turn off office hours
+    function officeHours() public override returns (bool) {
+        return false;
     }
 
-    function integrateCentrifugeCollateral(CentrifugeCollateralValues memory collateral) internal {
-        address MIP21_LIQUIDATION_ORACLE =
-            DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
-
-        address vat = DssExecLib.vat();
-
-        // Sanity checks
-        require(GemJoinAbstract(collateral.MCD_JOIN).vat() == vat, "join-vat-not-match");
-        require(GemJoinAbstract(collateral.MCD_JOIN).ilk() == collateral.ilk, "join-ilk-not-match");
-        require(GemJoinAbstract(collateral.MCD_JOIN).gem() == collateral.GEM, "join-gem-not-match");
-        require(GemJoinAbstract(collateral.MCD_JOIN).dec() == DSTokenAbstract(collateral.GEM).decimals(), "join-dec-not-match");
-
-        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(
-            collateral.ilk, collateral.PRICE, collateral.DOC, collateral.TAU
-        );
-        (,address pip,,) = RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).ilks(collateral.ilk);
-
-        // Set price feed for RWA-00x
-        DssExecLib.setContract(DssExecLib.spotter(), collateral.ilk, "pip", pip);
-
-        // Init RWA-00x in Vat
-        Initializable(vat).init(collateral.ilk);
-        // Init RWA-00x in Jug
-        Initializable(DssExecLib.jug()).init(collateral.ilk);
-
-        // Allow RWA-00x Join to modify Vat registry
-        DssExecLib.authorize(vat, collateral.MCD_JOIN);
-
-        // Allow RwaLiquidationOracle to modify Vat registry
-        // DssExecLib.authorize(vat, MIP21_LIQUIDATION_ORACLE);
-
-        // set ilk/global DC
-        DssExecLib.increaseIlkDebtCeiling(collateral.ilk, collateral.CEIL, true);
-
-        // No dust
-        // DssExecLib.setIlkMinVaultAmount(collateral.ilk, 0);
-
-        // stability fee
-        DssExecLib.setIlkStabilityFee(collateral.ilk, collateral.RATE, false);
-
-        // collateralization ratio
-        DssExecLib.setIlkLiquidationRatio(collateral.ilk, collateral.MAT);
+    function actions() public override {
 
-        // poke the spotter to pull in a price
-        DssExecLib.updateCollateralPrice(collateral.ilk);
+        // Update early RWA tokens names in ilk registry
+        IlkRegistryAbstract ILK_REGISTRY = IlkRegistryAbstract(DssExecLib.reg());
 
-        // give the urn permissions on the join adapter
-        // DssExecLib.authorize(collateral.MCD_JOIN, collateral.URN);
+        address join;
+        address gem;
+        uint8   dec;
+        uint96  class;
+        address pip;
+        address xlip;
+        string memory name;
+        string memory symbol;
 
-        // set up the urn
-        Hopeable(collateral.URN).hope(collateral.OPERATOR);
+        (, join, gem, dec, class, pip, xlip, name, symbol) = ILK_REGISTRY.ilkData("RWA001-A");
```

Comment:
Sorry for three separate deletes, the github interface was messed up and would only let me do one at a time.
The comment related to these deletes is in the last section.
```suggestion
```